packages/rich-text/src/getPastedBlocks.ts

```diff
@@ -47,20 +58,34 @@ class CustomBlockRenderMap {
       },
     };
   }
-
   // The only function from immutable.js#Map, that we need to implement
-  mapKeys(callback) {
-    return Object.keys(this._map).map((key) => callback(key, this._map[key]));
+  mapKeys<M>(
+    callback: (
+      key?: string | undefined,
+      value?: DraftBlockRenderConfig | undefined,
+      iter?:
+        | Immutable.Collection.Keyed<string, DraftBlockRenderConfig>
+        | undefined
+    ) => M
+  ): M[] {
+    return Object.keys(this._map).map((key) =>
+      callback(key, this._map[key as keyof TagMap])
+    );
   }
 }
 
-function getPastedBlocks(html, existingStyles = []) {
+// Overriding this since entityMap is marked as `any` in there, however, it seems to be an object in this case.
+type ConvertFromHTMLReturn = {
+  contentBlocks: ContentBlock[];
+  entityMap: Record<string, unknown>;
+};
+function getPastedBlocks(html: string, existingStyles: string[] = []) {
   const renderMap = new CustomBlockRenderMap();
   const { contentBlocks, entityMap } = convertFromHTML(
     html,
     undefined /* This has to be undefined to trigger default argument */,
     renderMap
```

Comment:
@barklund Not sure how to fix this TS error, from the `draft-js` [examples](https://draftjs.org/docs/advanced-topics-custom-block-render-map/), I can see that the `renderMap` could be created just by `Immutable.Map(mapObject)` instead of creating a custom class for it, however, I assume that it's not done like this currently not to require `immutabe`.

When using `Immutable.Map`, TS doesn't show any errors and it works, however, since it works currently as well then perhaps the types are just not correct in `draft-js`?

Let me know if you have any thoughts on how to fix this type.